packages/discord.js/typings/index.d.ts

```diff
@@ -3589,6 +3589,7 @@ export class VoiceState extends Base {
   public setRequestToSpeak(request?: boolean): Promise<this>;
   public setSuppressed(suppressed?: boolean): Promise<this>;
   public edit(options: VoiceStateEditOptions): Promise<this>;
+  public fetch(force?: boolean): Promise<this>;
```

Comment:
It doesn't return `this`
```suggestion
  public fetch(force?: boolean): Promise<VoiceState>;
```